config/label-management.md

```diff
@@ -0,0 +1,126 @@
+```
+
+1. Save your changes and run `source config`
+
+## Label management operations ##
+
+There are several operations you'll probably want to do, now that you've
+set up the environment and have the script ready.
+
+### Re-initializing the environment ###
+
+When you return to the directory in a new shell, remember to:
+
+```console
+source venv/bin/activate
+source config
+```
+
+### List all organization repositories ###
+
+```console
+python list-all-repos.py
+```
+
+This will yield a list of all repositories you have access to in the specified
+GitHub organization. If you have not specified a token, this will yield a list
+of all public repositories.
+
+### Back up all current repository labels ###
+
+This will generate a `.toml` file for each repository in a directory of your
+choosing. Each `.toml` file will contain the details of the repository's
+labels, including names, colors, and descriptions.
+
+Please note these output files are the same format as the `default-labels.toml`
+file, so you can use one as a template if you have a repository set up with
+the labels you want organization-wide.
+
+```console
+# Make the directory if it doesn't already exist
+# The -p option specifies to create intermediate directories as required
+mkdir -p repo-labels-backup
+
+# Use the output of list-all-repos.py to fetch each repository's labels
+python list-all-repos.py | xargs -I {} labels fetch --owner $LABELS_ORGNAME --repo {} --filename repo-labels-backup/{}-labels.toml
+```
+
+### Apply default labels across the organization ###
+
+View and edit `default-labels.toml` to specify the default labels and their
+attributes such as name, color, and description. This is the same format as
+the output `.toml` files from the "back up" step, so if you have an example
+repository, you can copy straight from that repository's file.
+
+Please note that the `labels ... sync` operation is **destructive** and will
+reduce the set of labels in each repository to _only_ those specified.
+
+The label history of each issue and PR will show the removal of the previous
+labels in case you need to restore some.
+
+Once you've set up your `default-labels.toml` file, create a list of all the
+repositories in a file so you can edit the list to remove any repositories you
+do not want to alter labels for.
+
+```console
+python list-all-repos.py > repolist.txt
+```
+
+Make any edits to the list to exclude repositories that are special or that
+you don't want to sync the labels for whatever reason, and then apply the
+labels to the remaining list of repositories.
+
+```console
+cat repolist.txt | xargs -I {} labels --verbose sync --owner $LABELS_ORGNAME --repo {} --filename default-labels.toml
+```
+
+#### Dangerous operation: Apply labels in one step ####
+
+The following is **not recommended**, but to apply the `sync` without an
+interstitial `repolist.txt` file (and live dangerously), you can instead run
+the `sync` in one step.
+
+```console
```

Comment:
```suggestion
```sh
```

Reply:
This is command-line usage so I believe `console` is more accurate.